docs/sources/explore/logs-integration.md

```diff
@@ -14,113 +14,112 @@ weight: 15
 
-Shows or hides the unique labels column that includes only non-common labels. All common labels are displayed above.
+### Log result meta information
 
-### Wrap lines
+Above the received log lines, you can find essential meta information, including:
 
-Set this to True if you want the display to use line wrapping. If set to False, it will result in horizontal scrolling.
+- **Number of received logs**: Indicates the total count of logs received for the current query or time range.
+- **Error**: Displays possible error in your log results
+- **Common labels**: Shows common labels.
+- **Total bytes processed**: Represents the cumulative size of the log data processed in bytes.
 
-### Prettify JSON
+> **Note:** The availability of certain meta information may depend on the data source, and as a result, you may only see some of these details for specific data sources.
 
-Set this to `true` to pretty print all JSON logs. This setting does not affect logs in any format other than JSON.
+### Escaping newlines
 
-### Deduplication
+Explore automatically detects some incorrectly escaped sequences in log lines, such as newlines (`\n`, `\r`) or tabs (`\t`). When it detects such sequences, Explore provides an "Escape newlines" option.
 
-Log data can be very repetitive and Explore can help by hiding duplicate log lines. There are a few different deduplication algorithms that you can use:
+To automatically fix incorrectly escaped sequences that Explore has detected:
 
-- **Exact -** Exact matches are done on the whole line except for date fields.
-- **Numbers -** Matches on the line after stripping out numbers such as durations, IP addresses, and so on.
-- **Signature -** The most aggressive deduplication, this strips all letters and numbers and matches on the remaining whitespace and punctuation.
+1. Click "Escape newlines" to replace the sequences.
+2. Manually review the replacements to confirm their correctness.
 
-### Display results order
+Explore replaces these sequences. When it does so, the option will change from "Escape newlines" to "Remove escaping". Evaluate the changes as the parsing may not be accurate based on the input received. You can revert the replacements by clicking "Remove escaping".
 
-You can change the order of received logs from the default descending order (newest first) to ascending order (oldest first).
+### Log level
 
-## Labels and detected fields
+For the logs where a `level` label is specified, we use the value of the label to determine the log level and update color of log line accordingly. If the log doesn't have a level label specified, we try to find out if its content matches any of the supported expressions (see below for more information). The log level is always determined by the first match. In case Grafana is not able to determine a log level, it will be visualized with an unknown log level.
```

Comment:
```suggestion
For the logs where a `level` label is specified, we use the value of this label to determine the log level and update color of each log line accordingly. If the log doesn't have an specificed level label, we try to find out if its content matches any of the supported expressions (see below for more information). The log level is always determined by the first match. In the case where Grafana is not able to infer a log level field, it will be visualized with an unknown log level.
```